pkg/deploy/api/types.go

```diff
@@ -153,24 +153,24 @@ type DeploymentList struct {
 const (
 	// DeploymentConfigAnnotation is an annotation name used to correlate a deployment with the
 	// DeploymentConfig on which the deployment is based.
-	DeploymentConfigAnnotation = "deploymentConfig"
+	DeploymentConfigAnnotation = "openshift.io/deployment-config"
 	// DeploymentAnnotation is an annotation on a deployer Pod. The annotation value is the name
 	// of the deployment (a ReplicationController) on which the deployer Pod acts.
-	DeploymentAnnotation = "deployment"
+	DeploymentAnnotation = "openshift.io/deployment"
 	// DeploymentPodAnnotation is an annotation on a deployment (a ReplicationController). The
 	// annotation value is the name of the deployer Pod which will act upon the ReplicationController
 	// to implement the deployment behavior.
-	DeploymentPodAnnotation = "pod"
-	// DeploymentStatusAnnotation is an annotation name used to retrieve the DeploymentStatus of
+	DeploymentPodAnnotation = "openshift.io/deployer-pod"
+	// DeploymentStatusAnnotation is an annotation name used to retrieve the DeploymentPhase of
```

Comment:
I like having what the annotation is expected to be on in the godoc comment... that's helpful